+bisos/+iteration/+step/convergence.m

```diff
@@ -0,0 +1,102 @@
+classdef convergence < bisos.iteration.Step
+   
+properties
+    type = 'convergence';
+    varout = [];
+    varin;
+    data_prev = true;
+    
+    levels;
+    
+    % default properties for convergence verification
+    ctol = 10^-9;
+    domain = [-1 1];
+end
+
+methods
+    function step = convergence(~, vars, varargin)
+        % new level set convergence setup
+        
+        for i = 1:length(vars)
+            if isa(vars{i}, 'char')
+                step.varin{length(step.varin)+1} = vars{i};
+            end
+        end
```

Comment:
I use a loop because I defined that the level sets may not be only defined by inner polynomials and scalars, in other words, the content in varargin its not only solutions in `sol`. 
In the demo example varargin is `{'V' 'g' p 'b'}` where p is just the unchangeable shape function. This loops just checks which variables from `sol` are needed to do convergence check.